File: backend/app_serverside/apps/todos/urls.py
from django.conf.urls.defaults import *
from piston.resource import Resource
from apps.todos.views import UserHandler, TodoHandler, MedicineHandler


user_handler = Resource(UserHandler)
todo_handler = Resource(TodoHandler)
medicine_handler = Resource(MedicineHandler)

urlpatterns = patterns('',
   url(r'^user/(?P<login>[^/]+)/(?P<password>[^/]+)/', user_handler),
   url(r'^users_by_role/(?P<role>[^/]+)/', user_handler),
   url(r'^users/', user_handler),
   url(r'^todos/', todo_handler),
   url(r'^get_todo/(?P<todo_id>[^/]+)/', todo_handler),
   url(r'^create_todo/(?P<user_id>[^/]+)/', todo_handler),
   url(r'^update_todo/(?P<todo_id>[^/]+)/(?P<user_id>[^/]+)/', todo_handler),
   url(r'^delete_todo/(?P<todo_id>[^/]+)/(?P<user_id>[^/]+)/', todo_handler),
   url(r'^medicines/', medicine_handler)
)

File: server/urls.js
var user = require('./routes/user');
var todo = require('./routes/todo');
var medicines = require('./routes/medicines');

exports.route = function(app, baseRoute) {
    app.get([baseRoute, 'user/:login/:password/'].join(''), user.getUser);
    app.get([baseRoute, 'users_by_role/:role/'].join(''), user.usersByRole);
    app.get([baseRoute, 'users/'].join(''), user.all);
    app.get([baseRoute, 'todos/'].join(''), todo.all);
    app.post([baseRoute, 'create_todo/:user_id'].join(''), todo.createTodo);
    app.put([baseRoute, 'update_todo/:todo_id/:user_id'].join(''), todo.updateTodo);
    app.delete([baseRoute, 'delete_todo/:todo_id/:user_id'].join(''), todo.deleteTodo);
    app.get([baseRoute, 'medicines/'].join(''), medicines.index);
};
